libs/zauth/src/Data/ZAuth/Token.hs

```diff
@@ -61,7 +71,16 @@ import Data.ByteString.Lazy.Char8 (split, break, drop)
 import Data.UUID
 import Sodium.Crypto.Sign (Signature (..))
 
-data Type = A | U | B | P deriving (Eq, Show)
+-- | A = Access (Used as short-lived token for Users)
+-- U = User (Used as a cookie for Users to refresh access tokens)
+-- B = Bot
+-- P = Provider
+-- LA = LegalHold Access (Used as short-lived token for LegalHold Service)
+-- LU = LegalHold User (Used as a cookie for LegalHold Service to refresh access tokens)
```

Comment:
now i'm wondering if `LD` (legalhold device) isn't a better name.  less symmetrical, but more accurate.  no strong opinion, though.

Reply:
you could also move the list from this comment into the declaration:

```haskell
data Type
    = A  --^ Access (...)
    | B  --^ ...
...
```

Reply:
> you could also move ...

thanks!

> now i'm wondering if LD (legalhold device) ...

I prefer the symmetrical name, I think.